config/zerolog.go

```diff
@@ -23,12 +25,17 @@ func init() {
 
 func InitZeroLog() {
 	if !zerologInitDone {
-		if ZeroLogJournalDEnabled {
-			log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+		var output io.Writer
+		output = os.Stderr
+		if !ZeroLogEnableJSONLogger {
+			output = zerolog.ConsoleWriter{Out: os.Stderr, NoColor: ZeroLogNoColor}
+		}
+		if ZeroLogJournalDDisabled {
+			log.Logger = log.Output(output)
 			log.Debug().Msg("Enabled console writer")
 		} else {
 			journalWriter := journald.NewJournalDWriter()
-			multi := io.MultiWriter(zerolog.ConsoleWriter{Out: os.Stderr}, journalWriter)
+			multi := io.MultiWriter(output, journalWriter)
```

Comment:
Some of the original logic is unnecessarily manual. In fact, logging both to output and to journald also seem a bit incorrect when the service is running under systemd, where output will go to journald, duplicating the logs.

It's possible to use [`Enabled()`](https://pkg.go.dev/github.com/coreos/go-systemd/v22/journal#Enabled) to determine if journald should be disabled and also [`StdoutIsJournalStream()`](https://pkg.go.dev/github.com/coreos/go-systemd/v22/journal#StdoutIsJournalStream) to detect if we're running as systemd service and disable plain stdout logging. Otherwise, it means service runs in foreground and logging to journald is likely not needed as the intent is to see the output.